Description: Do not install the Using Samba book when installing SWAT
 Using Samba is packaged in samba-doc, however upstream also
 installs it in SWAT install dirs
Author: Christian Perrier <bubulle@debian.org>
Forwarded: not-needed

Index: experimental/source3/script/installswat.sh
===================================================================
--- experimental.orig/source3/script/installswat.sh
+++ experimental/source3/script/installswat.sh
@@ -198,7 +198,11 @@
 
 # Install/ remove Using Samba book (but only if it is there)
 
-if [ "x$BOOKDIR" != "x" -a -f $SRCDIR../docs/htmldocs/using_samba/toc.html ]; then
+# Under Debian we don't actually install the book. The book is part of
+# the samba-doc package, so we just provide a symlink that points to
+# where the book is actually installed. The symlink is created in
+# debian/rules.
+if /bin/false; then
 
     # Create directories
 
